lib/quarry.rb

```diff
@@ -525,7 +525,7 @@ def load_config_file(name, slot)
 end
 
 def sync_chroot_repo
-  `sudo systemd-nspawn -q --bind-ro=#{INDEX_DIR}:#{CHROOT_QUARRY_PATH} -D #{CHROOT_ROOT_DIR} pacman -Sy`
+  `sudo systemd-nspawn -q --bind-ro=#{INDEX_DIR}:#{CHROOT_QUARRY_PATH} -D #{CHROOT_ROOT_DIR} pacman -Syu`
```

Comment:
The script stopped with

```
$ lib/update.rb 
:: Proceed with installation? [Y/n]
```

I believe it happened because of the change above. We need to tell pacman to accept default options and update packages quietly.